src/Common/Thermodynamics/relations.jl

```diff
@@ -1128,14 +1131,14 @@ function saturation_adjustment(
         _T_freeze::FT = T_freeze(param_set)
         e_int_upper = internal_energy_sat(
             param_set,
-            _T_freeze + sqrt(eps(FT)),
+            _T_freeze + temperature_tol,
```

Comment:
```suggestion
            _T_freeze + temperature_tol/2,
```
(so that the interval in the end is `temperature_tol` wide)

Reply:
Makes sense, good catch!